File: python/world_1/examples/3_usando_modulos/009_manipulando_texto/ex027_primeiro_nome_ultimo_nome/ex027.js
function haveNumber(variable) {

    if (variable.match(/[0-9]/)) {
        return true;
    } else {
        return false;
    }
}

function showName() {

    if (haveNumber(inputName.value) || (inputName.value).length == "") {

        divAnswer.innerHTML = `<div class="answer error">Informe um valor válido!</div>`;

    } else {

        divAnswer.innerHTML = `<div class="answer"></div>`;
        divAnswer.innerHTML += `<div class="answer">O nome completo informado foi ${inputName.value}</div>`;
        divAnswer.innerHTML += `<div class="answer">O 1º nome é ${inputNameSplitted[0]}</div>`;
        divAnswer.innerHTML += `<div class="answer">O último nome é ${inputNameSplitted[inputNameSplitted.length - 1]}</div>`;
        divAnswer.innerHTML += `<div class="answer">O nome completo tem ${inputNameSplitted.length} nomes.</div>`;

        console.log(inputNameSplitted)
    }

}

let inputName = document.querySelector('#input1');
let inputNameSplitted = inputName.value.split(" ");
let divAnswer = document.querySelector('.answer');

File: python/world_1/examples/3_usando_modulos/009_manipulando_texto/ex027_primeiro_nome_ultimo_nome/ex027.py
# Faça um programa que leia o nome completo de uma pessoa, mostrando em seguida o primeiro e o último nome separadamente.

# Ex: Ana Maria de Souza
# primeiro = Ana
# último = Souza

completeName = input("digite o seu nome completo: ")

completeNameSplitted = completeName.title().split()

if completeName.isnumeric():

    print('Informe um nome válido!')

else:

    print(f'O nome completo informado foi {completeName.title()}')
    print(f'O 1º nome é {completeNameSplitted[0]}.')
    print(f'O último nome é {completeNameSplitted[-1]}.')
